kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -85,6 +85,7 @@ public BigQuerySinkTask(BigQuery testBigQuery, SchemaRetriever schemaRetriever)
   @Override
   public void flush(Map<TopicPartition, OffsetAndMetadata> offsets) {
     try {
+      logger.info("Flushing {} records in the sink.", executor.getActiveCount());
```

Comment:
is this supposed to be executor.getActiveCount()?

Reply:
@mtagle Ah, I agree it's a bit vague and potentially confusing. (active + completed) tasks is incorrect since it would return a count of all tasks completed by this executor; and it's not possible to just get the "interval" of tasks since the last flush. I'm going to remove this line.